.github/workflows/build.yml

```diff
@@ -81,7 +81,9 @@ jobs:
       working-directory: tests
     - name: Print failure info
       run: |
-        for exp in *.exp;
+        set -- ./*.exp
+        [ -e "$1" ] || shit
```

Comment:
I don't understand the use of an obscenity here. And you could use `nullglob` as suggested.

Reply:
wow I'm so sorry about that

Reply:
A letter was missing: `shift`.